qa.gemspec

```diff
@@ -22,10 +22,13 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.6"
   s.add_dependency "activerecord-import"
   s.add_dependency "deprecation"
+  s.add_dependency "rdf", "2.2.3"
```

Comment:
This was supposed to be a temporary pin.  So glad you caught it.  There are two issues with 2.2.5.
1) rspec mocks fail - There is a fix for this.
2) rdf/rdf tries to read file contents with the wrong reader.  (https://github.com/ruby-rdf/rdf/issues/353)  This causes an empty graph to be returned.